src/components/ha-network.ts

```diff
@@ -0,0 +1,168 @@
+import "@polymer/paper-tooltip/paper-tooltip";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, state, property } from "lit/decorators";
+import {
+  Adapter,
+  NetworkConfig,
+  IPv6ConfiguredAddress,
+  IPv4ConfiguredAddress,
+} from "../data/network";
+import { fireEvent } from "../common/dom/fire_event";
+import { haStyle } from "../resources/styles";
+import { HomeAssistant } from "../types";
+import "./ha-checkbox";
+import type { HaCheckbox } from "./ha-checkbox";
+import "./ha-settings-row";
+import "./ha-icon";
+
+const format_addresses = (
+  addresses: IPv6ConfiguredAddress[] | IPv4ConfiguredAddress[]
+): TemplateResult => addresses.map(
+    (address) => html`<span>${address.address}/${address.network_prefix}</span>`
+  )
+
+const format_auto_detected_interfaces = (adapters: Adapter[]): TemplateResult | string => adapters.map((adapter) =>
```

Comment:
```suggestion
const format_auto_detected_interfaces = (adapters: Adapter[]): Array<TemplateResult | string> => adapters.map((adapter) =>
```

Reply:
Oops.  Pushed a different change before I saw this.  Reverted to this